tests/backend/test_interface.py

```diff
@@ -46,11 +47,14 @@ def receive(self):
     def send_raw(self, *args, **kwargs):
         self.mock.send_raw(*args, **kwargs)
 
+    def finger_touch(self, *args, **kwargs):
```

Comment:
Why add finger touch in this PR ? @lpascal-ledger

Reply:
Glitching